src/main/java/application/Routes.java

```diff
@@ -1,12 +1,15 @@
 package main.java.application;
 
 import main.java.server.Router;
+import main.java.server.response.Handler;
 
 import java.util.HashMap;
 import java.util.TreeMap;
 
 public class Routes {
-    private static TreeMap<String, HashMap<String, Handler>> ROUTES;
+    public static TreeMap<String, HashMap<String, Handler>> ROUTES;
```

Comment:
Is `ROUTES` used anywhere? Looks like `App` needs a router but not necessarily the routes themselves. Also, does `App` need to exist or could the process for starting up a server be simply creating a router and passing it to the server?

Reply:
No it's not used anywhere, and you're right about the `App` part, I'll refactor to have the router being passed into the server as an argument